jsk_arc2017_common/node_scripts/ekew_i_driver.py

```diff
@@ -23,12 +23,13 @@ def __init__(self):
             port, baudrate=2400, bytesize=7, parity=serial.PARITY_EVEN,
             timeout=timeout, writeTimeout=timeout)
         self.pub = rospy.Publisher('~output', WeightStamped, queue_size=1)
+        self.pub_raw = rospy.Publisher('~output_raw', WeightStamped, queue_size=1)
```

Comment:
is this for debugging?
if so, can you change topic name like `~debug/weight_raw`